NuGet.config

```diff
@@ -1,8 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <!-- Ignore global configuration -->
     <clear />
-    <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
-    <add key="OrchardCore" value="https://nuget.cloudsmith.io/orchardcore/preview/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
The feed is already there?!!

Reply:
Ooops, I didn't notice. I've reverted the name.

My point was just to remove the `OrchardCore` source. It's not even used and because of the central package management this kept giving `warning NU1507: There are 2 package sources defined in your configuration.`